thorlcr/activities/countproject/thcountprojectslave.cpp

```diff
@@ -167,12 +168,12 @@ class CountProjectActivity : public BaseCountProjectActivity, implements ILookAh
     }
     virtual void stop()
     {
-        PARENT::stop();
         if (first) // nextRow, therefore getPrevCount()/sendCount() never called
         {
             prevRecCount = count = getPrevCount();
             signalNext();
         }
+        PARENT::stop();
```

Comment:
This might be a good change, signalling earlier, potentially allowing next slave to continue quicker.
But don't know why it should have caused a deadlock.
Did you see a case where block in PARENT::stop() ? If so, why?